src/deviation.jl

```diff
@@ -81,26 +124,58 @@ function gkldiv{T<:AbstractFloat}(a::AbstractArray{T}, b::AbstractArray{T})
     return r::Float64
 end
 
+
 # MeanAD: mean absolute deviation
+"""
+    meanad(a, b) -> Float
+
+Return the mean absolute deviation between two arrays.
+"""
 meanad{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T}) = L1dist(a, b) / length(a)
 
+
 # MaxAD: maximum absolute deviation
+"""
+    maxad(a, b) -> Float
+
+Return the maximum absolute deviation between two arrays.
+"""
 maxad{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T}) = Linfdist(a, b)
 
+
 # MSD: mean squared deviation
+"""
+    msd(a, b) -> Float
+
+Return the mean squared deviation between two arrays.
+"""
 msd{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T}) = sqL2dist(a, b) / length(a)
 
+
 # RMSD: root mean squared deviation
+"""
+    rmsd(a, b, [normalize=<bool>]) -> Float
+
+Return the root mean squared deviation between two optionally
```

Comment:
I'd just say "between two arrays", and then mention the normalization feature; what kind of normalization is that?